pages/public_cloud/containers_orchestration/managed-rancher-service/getting-started/guide.en-gb.md

```diff
@@ -0,0 +1,275 @@
+---
+title: Getting Started with Managed Rancher Service
+excerpt: 'Find out how to start using Managed Rancher Service on OVHcloud'
+updated: 2024-02-01
+---
+
+> [!warning]
+>
+> Usage of [Managed Rancher Service](https://labs.ovhcloud.com/en/managed-rancher-service/) is currently in Alpha phase.
+> This guide may be incomplete and will be extended during the beta phase. Our team remains available on our dedicated Discord Channel, do not hesitate do join and reach us : <https://discord.gg/ovhcloud>. Ask questions, provide feedback and interact directly with the team that builds our Container and Orchestration services.
+>
+
+## Objective
+
+Container orchestration has become a cornerstone of modern application deployment, offering scalability, flexibility, and resource efficiency. Rancher is an open-source container management platform that simplifies the deployment and management of Kubernetes clusters.
+Managed Rancher Service by OVHcloud, provides a powerful platform for orchestrating Kubernetes clusters seamlessly. In this Geting Started guide we will explore the intricacies of setting up and managing containers clusters.
+
+
+## Rancher Creation and Access
+> [!warning]
+>
+> During the Alpha phase, Rancher access are directly provided by OVHcloud team. Manager implementation will be available at Beta release phase. Please ignore this part for now.
+>
+
+To initiate your journey, log in to the OVHcloud Manager and access the _Managed Rancher Service_ under the _Container & Orchestration_ section.
+Using the OVHcloud Manager you can trigger the creation of a Rancher which will be operated and managed by OVHcloud.
```

Comment:
We should mention somewhere that those configuration steps can also be done from the APIv2 (which is different from the regular https://api.ovh.com/console/#/cloud that customers are used to) => https://eu.api.ovh.com/console-preview/?section=%2FpublicCloud&branch=v2